WooCommerce/Classes/ViewRelated/Orders/Order Creation/EditableOrderViewModel.swift

```diff
@@ -180,7 +174,7 @@ final class EditableOrderViewModel: ObservableObject {
             purchasableItemsOnly: true,
             storageManager: storageManager,
             stores: stores,
-            supportsMultipleSelection: isProductMultiSelectionEnabled,
+            supportsMultipleSelection: true,
```

Comment:
Now that we removed single-selection, can we remove this parameter/property from the initializer/class?

Reply:
Yes! As this bit would involve modifying the initializer we also use for Coupons, we're tracking this on a different issue: https://github.com/woocommerce/woocommerce-ios/issues/9434